ajax/libs/x-tag/package.json

```diff
@@ -0,0 +1,27 @@
+{
+  "name": "x-tag",
+  "filename": "x-tag-core.min.js",
+  "description": "The X-Tag core library",
+  "version": "1.5.6",
+  "author": "csuwildcat",
+  "keywords": [
+    "x-tag",
+    "core",
+    "W3C"
+  ],
+  "license": "MIT",
+  "homepage": "http://www.x-tags.org",
```

Comment:
This url will redirect to ```http://x-tag.github.io/``` . I think we should use ```https://x-tag.github.io/``` here. 
It's better to use ```https``` instead of ```http```.